chain/client/src/stateless_validation/state_witness_tracker.rs

```diff
@@ -0,0 +1,213 @@
+use std::hash::{Hash, Hasher};
+use borsh::{BorshDeserialize, BorshSerialize};
+use lru::LruCache;
+use once_cell::sync::Lazy;
+
+use near_async::time::Clock;
+use near_o11y::metrics::{exponential_buckets, HistogramVec, try_create_histogram_vec};
+use near_primitives::sharding::ChunkHash;
+use near_primitives::stateless_validation::{ChunkStateWitness, ChunkStateWitnessAck};
+
+/// This is currently used for metrics computation, so we do not need to keep all the
+/// witnesses, so make the capacity 10.
+const CHUNK_STATE_WITNESS_MAX_RECORD_COUNT: usize = 5;
```

Comment:
`5` doesn't seem to match `10` mentioned in the comment.
Also I suggest increasing the value to something like 50 to make sure we can capture cases when the latency was very large. It should not be an issues since we keep pretty small amount of data per record.

Reply:
yes, ideally there should be few instances kept here, unless there is a very high latency. changed the code and comment.